src/main.rs

```diff
@@ -44,21 +44,21 @@ fn run_cmake_command(build_dir: &str, args: &[String]) -> io::Error {
 }
 
 fn main() -> io::Result<()> {
-    let mut args = Args::parse();
+    let Args { mut build_directory, args } = Args::parse();
```

Comment:
This makes both build_directory and args variables mutable?

Reply:
just `build_directory`